File: flutter_libs/ios/objectbox_flutter_libs/Sources/objectbox_flutter_libs/ObjectboxFlutterLibsPlugin.swift
import Flutter
import UIKit

public class ObjectboxFlutterLibsPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
      // Not using method channels, so not registering one.
      // Note: if ever implementing this, may have add workaround for https://github.com/flutter/flutter/issues/67624
  }

  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
    result(nil)
  }
}

File: flutter_libs/windows/objectbox_flutter_libs_plugin.cpp
#include "objectbox_flutter_libs_plugin.h"

namespace objectbox_flutter_libs {

// static
void ObjectboxFlutterLibsPlugin::RegisterWithRegistrar(
    flutter::PluginRegistrarWindows *registrar) {
  // Not using method channels, so not registering one.
  (void)registrar;
}

}  // namespace objectbox_flutter_libs
